tests/test_lite.sh

```diff
@@ -18,9 +20,28 @@ cleanup() {
         echo "killing webserver"
         kill $pid
     fi
+    if [ -n "${daemon_pid}" ] ; then
+      echo "killing docker daemon mock: "${daemon_pid}""
+      kill "${daemon_pid}"
```

Comment:
you sure this is getting called? I'd think the daemon_pid defined below was local to that function?

Reply:
Yep, it works.

```
104: + cleanup
104: + echo 'cleaning up temp dir'
104: cleaning up temp dir
104: + rm -rf /tmp/tmp.v4qKr9bzbl
104: + '[' -n 31 ']'
104: + echo 'killing webserver'
104: killing webserver
104: + kill 31
104: + '[' -n 16 ']'
104: + echo 'killing docker daemon mock: 16'
104: killing docker daemon mock: 16
104: + kill 16

```